scripts/motor-expert-screen

```diff
@@ -31,10 +31,21 @@ export EDMDATAFILES=.:${PCDS_EDMS}/xps8:/reg/g/pcds/package/epics/3.14/ioc/commo
 source /reg/g/pcds/setup/pathmunge.sh
 ldpathmunge /usr/local/lib
 
+# Check to see if this is a Beckhoff axis
+#   caget will return exit code 0 if :PLC:nErrorId_RBV exists, 
+#   we set isbeckhoff based on this.
+isbeckhoff=false
+caget "${PREFIX}:PLC:nErrorId_RBV" > /dev/null 2>&1
+if [ $? -eq 0 ]; then
+    isbeckhoff=true
+fi
+
 # Choose title so we can move the right window.
 # If this script hangs later, titles probably changed.
 rtyp=`caget -t $PREFIX.RTYP` > /dev/null 2>&1
-if [ "${rtyp}" == 'xps8p' ]; then
+if [ ${isbeckhoff} == true ]; then
+    title="Typhos Suite - ${PREFIX}"
```

Comment:
I think this is unused

Reply:
The title is used to reposition the new window under the mouse.

Reply:
Ah, gotcha! Somehow I thought this was being applied, rather it is set to match the truth.